### ISSUE TITLE
menu: popups do not allow clickthrough

### ISSUE THREAD

**Issue by clshortfuse** [clshortfuse/materialdesignweb#10]
Menus, currently, use an transparent scrim which is confusing for UX. If the scrim is not dimmed, then users will not know they are being blocked. Popups should close when:

* An outside element receives a click event
* An outside element receives focus
* An outside element receives a scroll event
* Window is resized (possibly)

Preliminary research shows this should be possible with a `EventListener` on `document` with the `capture: true` option.

The option of an invisible scrim may be optional, due to this being the default behavior in some environments (eg: Mac OS X), but it clickthrough should be the default implementation.

**Comment by clshortfuse** [comment]
fixed in bc924c3f10bfcef207b752adcd09f1d071c67ad3
